api/src/main/java/net/draycia/carbon/api/users/CarbonPlayer.java

```diff
@@ -58,13 +59,21 @@
      */
     Component createItemHoverComponent();
 
+    /**
+     * The player's locale.
+     *
+     * @return the player's locale, or null if offline
+     * @since 2.0.0
+     */
+    @Nullable Locale locale();
+
     /**
      * The player's selected channel. If none has been set, returns the server default.
      *
      * @return the player's selected channel
      * @since 2.0.0
      */
-    ChatChannel selectedChannel();
+    @Nullable ChatChannel selectedChannel();
```

Comment:
Javadocs says the default is returned if none is set, but the result is nullable, could probably do with clarifying

Reply:
Good catch